tests/restapi/test_restapi.py

```diff
@@ -208,6 +208,12 @@ def test_data_path(construct_url, vlan_members):
 This test creates a VNET. It adds routes to the VNET and deletes them
 '''
 def test_create_vrf(construct_url):
+    # Create Default VxLan Tunnel
```

Comment:
@prsunny I partially understand the idea to run these tests as a whole. 
I added re-creation of VXLAN tunnel in each test to address the following:
- Similar API requests in these tests neither contradict with nor depend on each other (e.g. if first test creates VLAN with id 1, the next test uses id 2 for VLAN. Even if at some point first test fails, other tests do not fail on VLAN creation as they use another id). This already makes tests almost completely independent from one another (except for VXLAN tunnel dependency from first test).
- While each test creates its unique VNET/VLAN etc., they share similar flow at some point. E.g., tests `test_data_path` and `test_create_interface` both follow pattern 'create VNET' -> 'create VLAN' -> 'add VLAN member' -> 'add VLAN neighbor' with a few minor differences (e.g., adding routes in first test and deleting each of created objects in the last test). So if we only wanted to check the behavior of deleting VLAN members/neighbors, running `test_create_interface` would be enough.
- If we can't create tunnel during first test, only the first test will state that there was an error with tunnel creation. The other tests will return errors with VNET creation, which looks misleading and slightly complicates debugging. 
- Also, DELETE request for VXLAN tunnel currently does not actually remove tunnel. This may cause problems if we want to add new tests with different tunnel configuration.

Reply:
@prsunny can you please take a look?

Reply:
@prsunny Thank you for pointing out that not all platforms may support tunnel deletion.
I have updated the tests according to your proposal - now instead of deleting tunnel between each test and re-creating it, tests will check if tunnel is present and create it if it was missing.

Reply:
If we are going to create a default VXLAN tunnel at the beggining of each test function, then I think we could create a pytest fixture to create a VXLAN tunnel if that doesn't exist and mark it as autouse?